src/components/ha-sidebar.ts

```diff
@@ -474,15 +474,14 @@ class HaSidebar extends LitElement {
 
       .menu {
         box-sizing: border-box;
-        height: 65px;
+        height: 64px;
         display: flex;
         padding: 0 12px;
-        border-bottom: 1px solid transparent;
         white-space: nowrap;
         font-weight: 400;
         color: var(--primary-text-color);
-        border-bottom: 1px solid var(--divider-color);
-        background-color: var(--primary-background-color);
+        border-bottom: 1px solid var(--divider-color, transparent);
+        background-color: var(--sidebar-header-background-color);
```

Comment:
I know I told you to remove it, but in the default theme, this var is not set and won't set a background now right? So we should add the fallback...

Reply:
Yep thats why i added it in the first place lol ;)
No problem... should i add it again? As i had it before?